Fix:#91 Improve introductory section (#104)
protofire/eth-cli: README.md

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## Installation

Install eth-cli globally:

> `npm install -g eth-cli`

Reviewer: ```suggestion
```shell
npm install -g eth-cli
```
```

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## Installation

> Install it globally:

Reviewer: ```suggestion
Install es-cli globally:
```

Reviewer 2: Just in case, it's `eth-cli`

`npm install -g eth-cli`

> You can also try it out with `npx`:

Reviewer: ```suggestion
You can also try es-cli out with `npx`:
```
